test/controllers/AdditionalMetadataNavigationControllerSpec.scala

```diff
@@ -163,12 +164,13 @@ class AdditionalMetadataNavigationControllerSpec extends FrontEndTestHelper {
       checkCommonFileNavigationElements(fileSelectionPageAsString, parentFolder, folderId, fileId, selectedFolderId)
       fileSelectionPageAsString.contains(
         s"""
-           |                                <button name="pageSelected" data-prevent-double-click="true" class="govuk-button__tna-button-link" type="submit" data-module="govuk-button" role="link" value="${currentPage - 1}">
+           |                                <button name="pageSelected" data-prevent-double-click="true" """.stripMargin +
```

Comment:
The same thing here. Can you have these two strings as part of the same multiline string? There's a couple more of these in the file if you can check them.